server-kotlin/src/main/kotlin/digital/cesko/Application.kt

```diff
@@ -29,6 +34,12 @@ class Application {
         filterRegistrationBean.setName("etagFilter")
         return filterRegistrationBean
     }
+
+    @Bean("fulltextIndex")
```

Comment:
Je toto spravny pristup? Potrebuji to injectovat do dvou objektu.

Reply:
imho ano

@lukas-krecan za Tebe to taky funguje?

Reply:
Jo, nebo se da pouzit `Quailfier`, ale je to jedno